### DISCUSSION TITLE
Interested in 2D visualization for small molecules

### DISCUSSION THREAD

**Discussion by Ashutosh-tec** [MDAnalysis/mdanalysis#4489] [GSoC Discussions]
Greetings! I'm Ashutosh, a passionate undergraduate student pursuing Data Science at IIT Madras. I bring valuable experience from my recent internship, where I successfully tackled computer vision and NLP tasks. Additionally, I've showcased my skills through the completion of several full-stack web development projects. I am confident that my involvement in 2D visualization for small molecules will contribute positively to the project. Looking forward to making a meaningful impact!

**Comment by Pratibhabhalerao080** [comment]
Hello, My name is Pratibha, I am pursuing my Master's in Data Science in New York. I am interested in 2D visualization for small molecules , want to contribute in this project.
